examples/example_android_opengl3/android/.gitignore

```diff
@@ -0,0 +1,12 @@
+.cxx
+.externalNativeBuild
+build/
+*.iml
+
+.idea
+.gradle
+local.properties
+/captures
+
+.DS_Store
```

Comment:
Removed these 3 (/captures is rarely created by Android Studio, but so are others as well...)
Added 2 lines to prevent committing the Gradle wrapper, that Android Studio generates all the time